terraform/modules/hub/hub_saml_proxy.tf

```diff
@@ -73,6 +73,7 @@ data "template_file" "saml_proxy_task_def" {
     certificates_config_cache_expiry = var.certificates_config_cache_expiry
     jvm_options                      = var.jvm_options
     memory_hard_limit                = var.saml_proxy_memory_hard_limit
+    service_log_level                = var.service_log_level
```

Comment:
Can we update the task definition JSON file to pick up this variable `service_log_level `?